qiskit_ibm_runtime/sampler.py

```diff
@@ -287,10 +285,9 @@ def _call(
             inputs=inputs,
             options=Options._get_runtime_options(combined),
         ).result()
-
         return SamplerResult(
-            quasi_dists=raw_result["quasi_dists"],
-            metadata=raw_result["metadata"],
+            quasi_dists=raw_result.quasi_dists,
+            metadata=raw_result.metadata,
         )
```

Comment:
If `raw_result` is already a `SamplerResult` object, the following perhaps works?

```suggestion
        return raw_result
```

Reply:
Yes that should work